pkg/apis/events/v1alpha1/storage_validation.go

```diff
@@ -29,9 +29,16 @@ func (current *Storage) Validate(ctx context.Context) *apis.FieldError {
 }
 
 func (current *StorageSpec) Validate(ctx context.Context) *apis.FieldError {
-	// TODO
 	var errs *apis.FieldError
 
+	if current.defaulterError != nil {
```

Comment:
Does the validation webhook chain inside of the mutation webhook?

Reply:
I think it does. Regardless, this doesn't work so it was reverted out.